src/amchart/Bar.js

```diff
@@ -16,22 +16,28 @@
     Bar.prototype = Object.create(CommonSerial.prototype);
     Bar.prototype.implements(INDChart.prototype);
     
+    /**
+     * Publish Params Common To Other Libraries
+     */   
     Bar.prototype.publish("paletteID", "Dark2", "set", "Palette ID", Bar.prototype._palette.switch());
```

Comment:
I know this is not part of this pull request, but its the wrong default palette.